app/presenters/task_presenter/uncompleted.rb

```diff
@@ -1,25 +1,26 @@
 module TaskPresenter
   class Uncompleted < SimpleDelegator
 
-    def completion_command
-      TaskCompletion.new(task_id: id)
-    end
-
-    def signed_up?(user)
-      assignees.include?(user)
-    end
-
-    def completed_on
-      nil
+    def timestamp
+      return unless deadline
+      I18n.l(deadline, format: :short)
     end
 
     def completion_checkbox_state
       nil
     end
 
-    def show_deadline
-      return unless deadline
-      I18n.l(deadline, format: :short)
+    def completion_form(view)
+      return '<p></p>'.html_safe unless can_complete?(view.current_user.id)
+      view.form_for(completion_command, url: view.complete_task_url(self), method: :put, remote: true) do |f|
```

Comment:
__[rubocop]__
* Lint/UnusedBlockArgument: Unused block argument - `f`. You can omit the argument if you don't care about it. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)